src/main/java/ru/yandex/practicum/filmorate/storage/dao/film/FilmDao.java

```diff
@@ -1,9 +1,16 @@
 package ru.yandex.practicum.filmorate.storage.dao.film;
 
+import ru.yandex.practicum.filmorate.model.Film;
 import ru.yandex.practicum.filmorate.storage.FilmStorage;
 
+import java.util.List;
+
 public interface FilmDao extends FilmStorage {
     boolean checkFilmExist(Integer id);
 
+    List<Film> findDirectorsFilmsSortedByRate(Integer directorId);
```

Comment:
Для единого стиля именования методов проекта.
```suggestion
    List<Film> getDirectorsFilmsSortedByRate(Integer directorId);
```